### ISSUE TITLE
Repeater field in email notification prevents the notification from being sent

### ISSUE THREAD

**Issue by Timusan** [verbb/formie#498]
**Description**

Having a repeater field in an email notification prevents the notification from being sent out, without any error.

**Steps to reproduce**
1. Create a form with a repeater field (in my case I put two date fields in there)
2. Create a notification and put the repeater field in there via the + sign and selecting the field
3. Enable the notification
4. Input a submission

No notification will be sent out.
Now remove the field from the notification and it will sent out.

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.7
- Craft version: 3.6.17
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.8](https://github.com/verbb/formie/releases/tag/1.4.8)
